tests/test_management_commands.py

```diff
@@ -360,27 +431,41 @@ def test_basic_invocation(self):
         num_docs = MovieSearchIndex.get_num_docs()
         self.assertEqual(num_docs, 2, "After updating the index, it should have had two documents in elasticsearch")
 
-        # destroy the indexes as well as the Index, IndexVersion, IndexAction objects
-        call_command('es_dangerous_reset')
+        hidden_index_name = ".hidden-index"
+        with get_new_search_index(hidden_index_name):
```

Comment:
TestEsDangerousResetManagementCommand: added a test to ensure that indexes that begin with `.` such as `.kibana` don't get wiped out by `./manage.py es_dangerous_reset`